_posts/2025-03-31-v5-1-latest-release.md

```diff
@@ -102,16 +102,22 @@ For the existing release lines, we will set the following phase dates:
 
 | Major | CURRENT | ACTIVE | MAINTENANCE | EOL |
 | ----- | ------- | ------ | ----------- | --- |
-| 4.x   |         |        | 2025-04-01 | 2026-10-01 or later |
-| 5.x   | 2024-09-11 | 2025-03-31 | 2026-04-01 | 2027-04-01 |
-| 6.x   | TBD after 2025-10-01 | | | |
+| 4.x   |         |        | 2025-04-01 | *no sooner than 2026-10-01 |
+| 5.x   | 2024-09-11 | 2025-03-31 | **no sooner than 2026-04-01 | **no sooner than 2027-04-01 |
+| 6.x   | ***no sooner than 2025-10-01 | | | |
```

Comment:
```suggestion
| 6.x   | ***no sooner than 2026-01-01 | | | |
```

this would be more in alignment with the minimum dates, because it reflects a 3-month `current` period before the earliest possible `active` start for v6.  also seems like releasing `6.0.0` six months from today seems ambitious but maybe not?

I don't have strong feelings on this change; if we really think there's a good chance we could ship that early (2025-10-01), then it's fine as is, but would also mean v6 is forced to stay in `current` for 6 months

Reply:
The date math I did was based off this line from the ADR:

> During the `ACTIVE` period, a new major version may be released but *not* until at least 6 months have passed
  since the `CURRENT` version became `ACTIVE`.

v5 starts `ACTIVE` today, April 1st 2025, so I added 6 months to that. I believe that is correct according to the language in the ADR, but I do agree there is a good chance we *wont* ship earlier than 2026. So if we want to make this chance so folks don't come looking for something sooner I am alright with that, but it would not technically be the minimum defined by the proposed policy. But again, is is much more likely to be accurate if we push the date further out lol, no argument there.

Reply:
I don't know how or where that text got written.  We agreed previously to:

> A new major version will be released a minimum of 12 months after the last major version release.

Reply:
Ah, ok I must have pulled it from an outdated one. I will update the ADR and then we can just merge suggested change I think?